lametro/models.py

```diff
@@ -153,6 +153,56 @@ def board_report(self):
 
         return br
 
+    @property
+    def actions_and_agendas(self):
+        '''
+        Return list of dictionaries, each representing an action or an agenda on
+        which the given bill appears, in the format:
+
+        {
+            'date': <datetime>,
+            'description': <str>,
+            'event': <LAMetroEvent>,
+            'organization': <Organization>
+        }
+        '''
+        actions = self.actions.all()
+        events = LAMetroEvent.objects.filter(agenda__related_entities__bill=self)
+
+        data = []
+
+        for action in actions:
+            event = LAMetroEvent.objects.filter(\
+                participants__entity_type='organization',\
+                participants__name=action.organization,
+                start_time__date=action.date)
+
+            
+            action_dict = {
+                'date': action.date_dt,
+                'description': action.description,
+                'event': event,
+                'organization': action.organization
+            }
+            
+            data.append(action_dict)
+
+        for event in events:
+            # Use a description of "SCHEDULED"
+            related_entity = self.eventrelatedentity_set.get(agenda_item__event=event)
```

Comment:
The event related entity associates the bill with an event agenda. You're right (and it's a bit confusing!) that `organization` is a possible related entity, but we don't define it, i.e., this is coming back as null in the template. Instead, you can get at the organization via `event.participants` (see [`EventParticipant`](https://github.com/opencivicdata/python-opencivicdata/blob/ad30a91a243fd8784955b2440e79f23ba6fe7512/opencivicdata/legislative/models/event.py#L143)).